src/libs/protocols/protos/agent.proto

```diff
@@ -505,3 +507,55 @@ message GetMetricsRequest {}
 message Metrics {
 	string metrics = 1;
 }
+
+message VolumeStatsRequest {
+	string path = 1;
```

Comment:
Is it a `guest path` or a `container path`?

Reply:
+1.

Not specific to this PR but I really wish _all_ protocol buffer messages were fully documented.

Reply:
It's the guest path. I have updated and added comments.